tests/neg/i5397.scala

```diff
@@ -0,0 +1,41 @@
+import annotation.tailrec
+
+object Test {
+  def foo(x: => Int) = 1
+  def bar(x: Int => Int) = 1
+
+  @tailrec def rec1: Int =
+    foo(rec1) // error not in tail position
```

Comment:
Isn't the convention to start with `error:`, including the colon?

Reply:
Indeed